app/views/admin/data_update_scripts/index.html.erb

```diff
@@ -14,8 +14,11 @@
         <tr>
           <td class="whitespace-nowrap"><%= script.id %></td>
           <td><%= script.file_name %></td>
-          <td class="whitespace-nowrap justify-content-center">
+          <td class="justify-content-center">
             <%= script.status %>
+            <% if script.error %>
```

Comment:
nitpick: if error is a string, this will be true even for empty strings (I know, Ruby is a PITA on these things :D)